admin-part/src/main/kotlin/Scope.kt

```diff
@@ -60,8 +60,18 @@ class ActiveScope(
 
     val activeSessions = AtomicCache<String, ActiveSession>()
 
-    val methodsCoveredByTestCache = getCache<Int, MethodsCoveredByTest>(realtimeCalculationCache)
-
+    /**
+     * The cache is used for real-time coverage calculations.
+     * Enabling and disabling depends on the system property: "drill.plugins.test2code.features.realtime.cache".
+     * Only tests in the finalize state are added to the cache
+     */
```

Comment:
duplication of comment is not well practice, can we avoid it?

Reply:
Hmm, i think we can if move it to getCache method